actions/views/threads.ts

```diff
@@ -41,14 +37,10 @@ export function manuallyMarkThreadAsUnread(threadId: string, lastViewedAt: numbe
     ]);
 }
 
-export function switchToGlobalThreads() {
-    return (_dispatch: DispatchFunc, getState: GetStateFunc) => {
-        const state = getState() as GlobalState;
-        const teamUrl = getCurrentRelativeTeamUrl(state);
-        browserHistory.push(`${teamUrl}/threads`);
+export function switchToGlobalThreads(teamUrl: string) {
+    browserHistory.push(`${teamUrl}/threads`);
```

Comment:
Although yes it we can pass in the team url as parameters or get it from the state, both are correct approach here. But in context on this pr i think lets stick with the original implementation is getting the relativeteamurl from state and take it from there.